client/src/components/Users.jsx

```diff
@@ -0,0 +1,23 @@
+import React, { Fragment } from 'react';
+
+import User from './User.jsx';
+
+
+const Users = props => {
+    return (
+        <Fragment>
+            <h3>Users</h3>
+
+            {props.users.map(user => {
```

Comment:
I'd also account for no users (or more realistically, they haven't loaded yet) with a simple loading/retrieving message while they wait